### ISSUE TITLE
Main image in html is only saving but not the other images

### ISSUE THREAD

**Issue by Hemachandra-Reddy** [JonasCz/save-for-offline#21]
Hi,
Thanks a lot for your code.
In my HTML page which i am trying to save offline, There is one big image and some share icons like fb,twitter...and at the end of the page there are some small images.
When i save this HTML everything is fine except the small images at the end of the page are being replaced with the main big image.
Please help me to fix this issue.

**Comment by JonasCz** [comment]
I'm aware of a similar problem, and I'm working on a fix. Sorry for somehow not seeing this :-/

**Comment by Hemachandra-Reddy** [comment]
Hi Jonas,
Thanks for the reply, 
I found the problem, All the images in the HTML are having the same name "image.jpg", but the path is different, In the code actually the path is being trimmed and only image name is being considered for saving, so the images are being overridden, I changed the code to use the full path as image name and its working absolutely fine.

**Comment by JonasCz** [comment]
Yep, That's the exact problem - It also screws saving articles from Wikipedia, as all the formatting is missing, due to wikipedia using the same filename for all the CSS and scripts. Still trying to find a good fix for this. Again, sorry for not seeing your issue.
